src/enum_net.cpp

```diff
@@ -980,6 +980,7 @@ namespace libtorrent
 		GetIfEntry_t GetIfEntry = (GetIfEntry_t)GetProcAddress(iphlp, "GetIfEntry");
 		if (!GetIfEntry)
 		{
+			FreeLibrary(iphlp);
```

Comment:
basically, I think the proper solution here would be to load the library once, into a static variable and then not load it again. the same should probably be done for the actual functions we pull out of the library too.
If we need to free the library, it should really be done as a scope_guard or handle, that calls ``FreeLibrary()`` and the end of the scope